src/timestamp.rs

```diff
@@ -434,6 +478,10 @@ pub mod context {
             // where the clock sequence doesn't change regardless of the timestamp
             self.count.fetch_add(1, Ordering::AcqRel) & (u16::MAX >> 2)
         }
+
+        fn usable_bits(&self) -> usize {
+            14
```

Comment:
We need to add another implementation here with a 42bit counter that's intended to be used by v7 UUIDs.